sources/packages/backend/libs/integrations/src/services/sfas/sims-to-sfas.model.ts

```diff
@@ -1,6 +1,18 @@
-import { Student } from "@sims/sims-db";
+import { OfferingIntensity, Student } from "@sims/sims-db";
 
 export type StudentDetail = Student & {
   cslfOverawardTotal?: string;
   bcslOverawardTotal?: string;
 };
+
+export type ApplicationRecord = {
```

Comment:
Minor. We generally use interface unless we are deriving a type from existing model.